packages/dd-trace/src/tracer.js

```diff
@@ -19,6 +20,7 @@ class DatadogTracer extends Tracer {
 
     this._scope = new Scope()
     setStartupLogConfig(config)
+    this.appsec = new AppsecSDK(this, config)
```

Comment:
We've to change it after the merge of https://github.com/DataDog/dd-trace-js/pull/2698

Reply:
you mean place it in the constructor?

Reply:
place it in the proxy instead of `DatadogTracer`

Reply:
ok, then I would need to inject the `AppsecSDK` object as dependency to `DatadogTracer` constructor in order to be able to call `setUser` function without duplicating the `AppsecSDK` object, e.g.: 
``
this.appsec = new AppsecSDK(this, config)
this._tracer = new DatadogTracer(this.appsec, config)     
``
On the other hand I could refactor AppsecSDK to be a singleton so we can get several instances without duplicating it.
WDYT?